synchro/synchrotest.py

```diff
@@ -131,6 +131,10 @@
     "TestCursor.test_comment",
     "TestCursor.test_where",
     "TestGridfs.test_gridfs_find",
+    # Uses patching on grid_file._UPLOAD_BUFFER_SIZE which fails in synchro.
```

Comment:
Do we need to attach a PYTHON TODO jira ticket to this?

Reply:
No I see it as not worth the effort.